### ISSUE TITLE
Do not remove v1 documentation

### ISSUE THREAD

**Issue by NiloCK** [caddyserver/website#30]
Bug: v1 documentation will be removed "soon".

![image](https://user-images.githubusercontent.com/10780590/80262358-6ba68b00-864a-11ea-8835-559f8338ca2f.png)

Expected behavior: Documentation is permanently available. Failing that, availability should be predictable.

Fix: Remove scare banner from documentation pages, failing that, add a date so that users will have a firmer call-to-action to make a local copy.

**Comment by sorbits** [comment]
I would appreciate if the title of v1 documentation could be updated to include the version number.

For example searching for `caddy redirect` I currently get this shown as first result:

    caddyserver.com › docs › redir
    http.redir - Caddy User Guide

I propose adding `v1` to the title so that it is immediately clear to the user (of the search engine) that this result pertains to  the previous version of Caddy.

**Comment by mohammed90** [comment]
> For example searching for `caddy redirect` I currently get this shown as first result:
> 
>     caddyserver.com › docs › redir
>     http.redir - Caddy User Guide
> 
> I propose adding `v1` to the title so that it is immediately clear to the user (of the search engine) that this result pertains to  the previous version of Caddy.
> 

For the record, this navigation flow doesn't lead to a page related to v1 at all.

**Comment by sorbits** [comment]
> For the record, this navigation flow doesn't lead to a page related to v1 at all.

Are you claiming that the URL in the above search result is not related to v1? Because it most certainly is, it even has a temporary (307) redirect to a URL which makes v1 explicit:

    curl -sI https://caddyserver.com/docs/redir|grep -i location
    location: /v1/docs/redir

Or are you saying that you do not get the same v1 result when searching for `caddy redirect`? That could very well be, but would still be nice for those of us who do get v1 documentation among our search results, to easily see from the title of these results, that they are for v1.

**Comment by francislavoie** [comment]
I think the confusion stemmed from the fact you didn't specify that you're searching with Google, which decided it made sense to replace `/` in URLs with ` › ` for some reason. 

Typically that character is used when explaining instructions in plain english like: "go to `caddyserver.com` then go to `docs` then go to `redir`".

**Comment by francislavoie** [comment]
The v1 docs have been preserved as a downloadable archive (link appears at the bottom of the sidebar of the v2 docs): https://caddyserver.com/caddy-v1-docs-archive.tar.gz

**Comment by polarathene** [comment]
> I propose adding `v1` to the title so that it is immediately clear to the user (of the search engine) that this result pertains to the previous version of Caddy.

Initially I was googling how to do things with Caddy and when I clicked several results that went to official docs they failed to load a page, seemed like the site might have been down, I tried the root domain though, and it worked fine, noticing that I can't access v1 docs.

This was unexpected to have to search results sending me to docs that don't exist anymore, considering what Caddy does, I'm surprised that removing the v1 docs didn't account for re-routing users to the root path of v2 docs, or better, a deprecation notice page that v1 docs have been removed and they should instead view the v2 docs, while providing the v1 doc archive link.

Personally, since I saw a community thread on the forum link to v1 docs posted just 30 days ago, I was surprised that I couldn't look at what had been discussed, even if I don't intend to use v1 of Caddy, I would have appreciated being able to view those specific docs search results or communities had linked to, as v2 seems too fresh to have built up decent links. A clear warning similar to the image that indicates v1 docs are old and unsupported would be sufficient to avoid any confusion.

Are most of the v1 users migrating to v2? A download archive for docs seems like a hassle, v1 docs had their own route, what problems were they causing to be removed so soon? Were they causing support/maintenance issues or high traffic/bandwidth costs?

**Comment by emersion** [comment]
As an annoying workaround, you can access old docs here: https://web.archive.org/web/20200506105152/https://caddyserver.com/v1/

**Comment by mholt** [comment]
> This was unexpected to have to search results sending me to docs that don't exist anymore,

Our search engine? Or Google? Because we can't control what Google or any other search engine does. We even have pretty limited control over the Algolia site search too.

@emersion 

> As an annoying workaround, you can access old docs here: https://web.archive.org/web/20200506105152/https://caddyserver.com/v1/

Sure, but we've preserved the docs for the time being. As [@francislavoie said above](https://github.com/caddyserver/website/issues/30#issuecomment-661696098), there's an archive of them on our server as well.

@polarathene

> Were they causing support/maintenance issues or high traffic/bandwidth costs?

Yes. Very much yes.

**Comment by strass** [comment]
for those finding this on google, someone has the site deployed at https://caddy.its-em.ma/v1/

**Comment by bledari** [comment]
This is ridiculous. I guess Caddy is a joke then
